discovery-infra/test_infra/helper_classes/cluster_host.py

```diff
@@ -0,0 +1,50 @@
+import json
+from typing import List
+
+from assisted_service_client import Inventory, Host, Interface
+
+DEFAULT_HOSTNAME = "localhost"
+
+
+class ClusterHost:
+    def __init__(self, host_model: Host):
+        self.__hostModel = host_model
+        self.__inventory = Inventory(**json.loads(self.__hostModel.inventory))
+
+    def get_id(self):
+        return self.__hostModel.id
+
+    def get_inventory(self) -> Inventory:
+        return self.__inventory
+
+    def get_hostname(self) -> str:
+        return self.__hostModel.requested_hostname if self.__hostModel.requested_hostname else self.__inventory.hostname
+
+    def has_hostname(self) -> bool:
+        hostname = self.get_hostname()
+        return hostname and hostname != DEFAULT_HOSTNAME
+
+    def interfaces(self) -> List[Interface]:
+        return [Interface(**interface) for interface in self.__inventory.interfaces]
+
+    def macs(self) -> List[str]:
+        return list(map(lambda interface: interface.mac_address.lower(), self.interfaces()))
```

Comment:
can you transform to list-comprehension?

Reply:
I have changed it to:
`return [map(lambda interface: interface.mac_address.lower(), self.interfaces())]`
and got:
Expected type 'List[str]', got 'List[Iterator]' instead